Libplanet.Tests/Store/StoreExtensionTest.cs

```diff
@@ -91,11 +91,18 @@ public void LookupStateReference(StoreFixture fx)
 
         [Theory]
         [MemberData(nameof(StoreFixtures))]
-        public void ListAllStateReferences(StoreFixture fx)
+        public async Task ListAllStateReferences(StoreFixture fx)
         {
             Address address1 = fx.Address1;
             Address address2 = fx.Address2;
-            Address address3 = new PrivateKey().PublicKey.ToAddress();
+            Address address3 = fx.Address3;
+
+            var store = new DefaultStore(null);
+            var chain = TestUtils.MakeBlockChain(new NullPolicy<DumbAction>(), store);
+
+            var block1 = await chain.MineBlock(fx.Address1);
```

Comment:
Why we should use `chain.MineBlock()` instead of `TestUtils.MineNext()` here?

Reply:
There is no big meaning, but I wanted to mine block and to append automatically.
Because I thought there was no difference between calling `chain.Append()` after `TestUtils.MineNext()` and calling `chain.MIneBlock()`.